### PR TITLE
[FAI-6768] Add destination packaging test to ci.yml

### PR DESCRIPTION
**Pull request by JoshSauter**
## Description
Issues like the one which happened when we bumped Commander package which [had to be reverted](https://github.com/faros-ai/airbyte-connectors/pull/1063) might be able to be caught before merging by attempting the destination packaging within our CI workflow

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -74,6 +74,13 @@ jobs:
               ./scripts/source-acceptance-test.sh $source
             fi
           done
+      - name: Destination packaging test
+        run: |
+          cd destinations/airbyte-faros-destination && \
```
Comment (tovbinm) [inline .github/workflows/ci.yml:79]: I think `cd destinations/airbyte-faros-destination && npm i` should be enough

**Review by tovbinm — APPROVED** [review]

**Review by tovbinm — CHANGES_REQUESTED** [review]
hmm, sec...

**Comment by tovbinm** [comment]
I think we should create a new job and invoke it on PRs only.

**Review by tovbinm — COMMENTED** [review]
@JoshSauter @cjwooo please have a look

**Review by tovbinm — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -118,5 +118,49 @@ jobs:
...
+      - name: Test Airbyte CDK package
+        run: |
+          cd faros-airbyte-cdk && \
+          npm i && \
+          npm version 0.1.0 && \
+          npm publish --dry-run
+
+      - name: Test Airbyte Common package
+        run: |
+          cd faros-airbyte-common && \
```
Comment (JoshSauter) [inline .github/workflows/ci.yml:156]: I assume each of these steps starts from the base directory and not from the previous step?

**Review by JoshSauter — COMMENTED** [review]
LGTM assuming my comment is accurate (can't approve since it's technically my PR)

**Comment by JoshSauter** [comment]
The new step is failing with:
`npm ERR! notarget No matching version found for faros-airbyte-cdk@0.0.1.`

File: .github/workflows/ci.yml (hunk 2 above)
Comment (tovbinm) [reply .github/workflows/ci.yml:156]: yes

**Pull request merged**